actions/concurrent_runner.go

```diff
@@ -74,6 +78,24 @@ func (r *ConcurrentRunner) Start(a Action, timeout time.Duration) {
 		r.l.Errorf("Ignoring Start: %s.", err)
 		return
 	}
+	actionID, actionType := a.ID(), a.Type()
+
+	var requirements []lock.Entity
+	switch actionType {
+	case "pbm-switch-pitr":
```

Comment:
let's try to improve this switch statement to not to use string literal, e.g. define some string constant in the actions package and use it here and in the `Type() string` function.

Reply:
https://github.com/percona/pmm-agent/pull/285#discussion_r701123746

I tried to keep the scope of this PR small. Will do that.

Reply:
Removed at all